MinimedKit/PumpEvents/Sara6EPumpEvent.swift

```diff
@@ -27,7 +27,7 @@ public class Sara6EPumpEvent: TimestampedPumpEvent {
     
     let dateComponents = TimeFormat.parse2ByteDate(availableData, offset: 1)
     validDateStr = String(format: "%04d-%02d-%02d", dateComponents.year, dateComponents.month, dateComponents.day)
-    timestamp = TimeFormat.midnightForDate(dateComponents)
+    timestamp = dateComponents
```

Comment:
I agree it should have one day added, but it shouldn't use the local time zone midnight. Want me to change that?